app/models/educator.rb

```diff
@@ -47,6 +47,9 @@ def students_for_school_overview(*additional_includes)
       school.students
             .where(grade: grade_level_access)
             .includes(eager_loads)
+    else
+      logger.warn("fell through #students_for_school_overview for educator_id: #{self.id}")
```

Comment:
@alexsoble This looks like a bug to me.  This would fall through to a `nil` before, which came up in some of my manual testing.  I was curious how prevalent this was, and found some differences here between staging and production, so this could use some more eyes, maybe we can chat more Tuesday.